src/redux/categories/operations.js

```diff
@@ -0,0 +1,42 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const fetchCategories = createAsyncThunk(
+  'categories/fetchFirstCategories',
+  async (_, thunkAPI) => {
+    const state = thunkAPI.getState();
+    const { query } = state.categories;
+    try {
+      const response = await axios.get(`/getAllCategoriesList?query=${query}`);
+      const file = response.data.data.file;
```

Comment:
Are you sure about that ❓

Reply:
No. All the other operations aren't finished, like I said on comment before yours I only made login and register that works with backend because backend isn't yet finished with other endpoints. When I'm done with backend I'll start implement that to redux.

Reply:
Super 🙌